bundles/org.openhab.binding.enturno/.gitignore

```diff
@@ -0,0 +1,49 @@
+/target/
```

Comment:
A lot of files are already ignore by the root gitignore file.
Please remove this one and test again, there should be no file apart from maybe Idea files that are not already ignored.

Reply:
Done